test/testdata/parser/error_recovery/missing_operator.rb

```diff
@@ -0,0 +1,231 @@
+# typed: false
+
+module Opus::Log
+  def self.info; end
+end
+
+class A
+  def g
+    t = T.let(true, T::Boolean)
+    f = T.let(false, T::Boolean)
+    puts(0 .. 0)
+    puts(0 .. )
+    if 0 ..
+    end # error: unexpected token "end"
+    if 0 .. # error: Unsupported node type `IFlipflop`
+      #  ^^ error: missing arg to ".." operator
+      puts 'hello'
+    end
+    puts(1 ... 1)
+    puts(1 ... )
+    if 1...
+    end # error: unexpected token "end"
+    if 1... # error: Unsupported node type `EFlipflop`
+      # ^^^ error: missing arg to "..." operator
+      puts 'hello'
+    end
+    puts(2 + 2)
+    puts(2 + ) # error: missing arg to "+" operator
+    if 2 + # error: missing arg to "+" operator
+    end
+    if 2 + # error: missing arg to "+" operator
+      puts 'hello'
+    end
+    puts(3 - 3)
+    puts(3 - ) # error: missing arg to "-" operator
+    if 3 - # error: missing arg to "-" operator
+    end
+    if 3 - # error: missing arg to "-" operator
+      puts 'hello'
+    end
+    puts(4 * 4)
+    puts(4 * ) # error: missing arg to tSTAR2 operator
+    if 4 * # error: missing arg to tSTAR2 operator
+    end
+    if 4 * # error: missing arg to tSTAR2 operator
```

Comment:
I don't have a great idea for how to fix them. Do you?

The error messages are coming because bison lets you associate a string with a token, but Bison only lets you associate a given string with one token (because those strings like `"*"` can be used in production rules as well as appear in error messages).

Reply:
Extending https://github.com/sorbet/sorbet/blob/aef1b01d50e21dba800ec02dd4531d75cccd8093/parser/parser/cc/driver.cc#L15-L29

and using it in the appropriate places seems like it would do the trick?

Reply:
oh neat! i'll have a PR up for this once my stack lands so that i don't conflict with myself